postman/Get Clicks.postman_collection.json

```diff
@@ -0,0 +1,596 @@
+									"});",
+									"",
+									"pm.test(\"Successfully get an account Id\", function(){",
+									"    pm.response.to.have.status(200);",
+									"    var jsonData = pm.response.json();",
+									"    pm.expect(jsonData.entries.length).to.be.above(0);",
+									"    pm.globals.set(\"accountId\",jsonData.entries[0].id ) ;   ",
+									"});",
+									""
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "GET",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": ""
+						},
+						"url": {
+							"raw": "{{base_url}}/accounts",
+							"host": [
+								"{{base_url}}"
+							],
+							"path": [
+								"accounts"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "Get Lists on Account",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "4d22c8fa-8695-4098-a275-10658e8aaea5",
+								"exec": [
+									"pm.test(\"Successfully got a  random list\", function() {",
+									"    var jsonData = pm.response.json();",
+									"    pm.expect(jsonData.total_size).to.be.a('number')",
+									"    var list_entry = Math.floor((Math.random() * jsonData.total_size ))",
+									"    pm.globals.set(\"listId\",jsonData.entries[list_entry].id )",
+									"});"
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "GET",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": ""
+						},
+						"url": {
+							"raw": "{{base_url}}/accounts/{{accountId}}/lists",
+							"host": [
+								"{{base_url}}"
+							],
+							"path": [
+								"accounts",
+								"{{accountId}}",
+								"lists"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "Get Random List",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "b41770d3-1044-48f5-9536-6cf9ba89088a",
+								"exec": [
+									""
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "GET",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": ""
+						},
+						"url": {
+							"raw": "{{base_url}}/accounts/{{accountId}}/lists/{{listId}}",
+							"host": [
+								"{{base_url}}"
+							],
+							"path": [
+								"accounts",
+								"{{accountId}}",
+								"lists",
+								"{{listId}}"
+							]
+						}
+					},
+					"response": []
+				}
+			]
+		},
+		{
+			"name": "Get Broadcast Open Clicks",
+			"item": [
+				{
+					"name": "Get Total Sent Broadcasts",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "e238f70c-3010-41a2-ba60-6fd4e312850d",
+								"exec": [
+									"pm.test(\"Sucessfully got total number of broadcasts\", function(){",
+									"    var jsonData = pm.response.json();",
+									"    if (pm.expect(jsonData.total_size).to.be.above(0)) {",
```

Comment:
You don't need to check the return value of this assertion. If it fails, the test will stop.